api/admin/routes.py

```diff
@@ -1127,7 +1127,37 @@ def suppressed():
 @returns_json_or_response_or_problem_detail
 @requires_admin
 def genres():
-    """Returns a JSON representation of complete genre tree."""
+    """Returns a JSON representation of complete genre tree.
+    ---
+    get:
+      tags: 
+        - works
```

Comment:
Every other endpoint in this tag concerns fetching/updating an individual work, so I'm not sure this fits here, but not sure where else to put it... `administration` is getting pretty big. Maybe we could make a new one `standard lists` or something like that? It would include:
- this
- `admin/languages`
- `admin/roles`
- `admin/rights_status`
- `admin/media`
- anything else?

Reply:
Nothing else I have come across so far.

Reply:
cool, let's shift them into a category called `taxonomies`. That might be a touch inaccurate, but I think it captures what we want to convey to people!